### PR TITLE
Add snapshot tests to some campaign components

### PR DESCRIPTION
**Pull request by eseliger**

### CODE REVIEW

File: web/src/enterprise/campaigns/detail/BurndownChart.test.tsx [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+                                total: 10,
+                                open: 9,
+                            },
+                            {
+                                __typename: 'ChangesetCounts',
+                                closed: 0,
+                                date: '2019-11-14T23:17:24Z',
+                                merged: 0,
+                                openApproved: 0,
+                                openChangesRequested: 0,
```
Comment (mrnugget) [inline web/src/enterprise/campaigns/detail/BurndownChart.test.tsx]: I would avoid `0` here and in the other counts (`merged`, `openApproved`) to make sure to exercise every graphical layer of the chart.

File: web/src/enterprise/campaigns/detail/BurndownChart.test.tsx [hunk 2]
```diff
@@ -0,0 +1,50 @@
+import * as H from 'history'
+import React from 'react'
+import renderer from 'react-test-renderer'
+import { CampaignBurndownChart } from './BurndownChart'
+
+function createNodeMock() {
+    const doc = document.implementation.createHTMLDocument()
```
Comment (felixfbecker) [inline web/src/enterprise/campaigns/detail/BurndownChart.test.tsx]: I don't see this done in any other snapshot test in web, why is it needed? Does the SVG rendering not work without a real document?

File: web/src/enterprise/campaigns/detail/__snapshots__/BurndownChart.test.tsx.snap [hunk 3]
```diff
@@ -0,0 +1,28 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`CampaignBurndownChart renders the chart 1`] = `
```
Comment (felixfbecker) [inline web/src/enterprise/campaigns/detail/__snapshots__/BurndownChart.test.tsx.snap]: This snapshot seems rather not useful without the SVG in it?

File: web/src/enterprise/campaigns/detail/changesets/ChangesetNode.test.tsx [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+    IExternalChangeset,
+} from '../../../../../../shared/src/graphql/schema'
+import { setLinkComponent } from '../../../../../../shared/src/components/Link'
+
+describe('ChangesetNode', () => {
+    const history = H.createMemoryHistory({ keyLength: 0 })
+    const location = H.createLocation('/campaigns')
+    test('renders a changesetplan', () => {
+        setLinkComponent((props: any) => <a {...props} />)
+        afterAll(() => setLinkComponent(null as any)) // reset global env for other tests
```
Comment (felixfbecker) [inline web/src/enterprise/campaigns/detail/changesets/ChangesetNode.test.tsx]: Put this in a `beforeEach()`?

**Pull request merged**